cmake_ocl.sh

```diff
@@ -43,6 +48,7 @@ usage ()
 	echo -e "\t-k\t Build KNL variant.";
 	echo -e "\t-a\t Build KNC accelerator variant.";
 	echo -e "\t-g\t Build GPU variant.";
+	echo -e "\t-d\t Enable debug mode.";
```

Comment:
Maybe call it enable verbose, non-parallel build, as we are still doing a release build without debug symbols.
```
Enable verbose make for debugging the build process.
```